src/cloudsc_fortran_pyiface/pythonsrc/dwarfdriver.py

```diff
@@ -0,0 +1,83 @@
+"""
+Driver that:
+- loads input variables and parameters from .h5 file,
+- invokes Fortran kernel computation,
+- validates against reference results read from another .h5 file.
+"""
+import sys
+from pathlib import Path
+from operator import itemgetter
+import cloudsc as clsc
+from cloudsc_data import define_fortran_fields
+from cloudsc_data import load_input_fortran_fields, load_input_parameters
+from cloudsc_data import load_reference_fields
+from cloudsc_data import convert_fortran_output_to_python
+from cloudsc_data import cloudsc_validate
+sys.path.append('../../build/src/cloudsc_fortran_pyiface')
+sys.path.append('../../build/lib')
+sys.path.append('.')
+
+NPROMA=100
+NUMOMP=1
+NLEV=137
+NGPTOT=100
+NGPTOTG=100
```

Comment:
We should add some CLI interface for the runner here to vary NUMOMP and NPROMA, but that can be left for a follow-on PR. I recomment using [click](https://click.palletsprojects.com/en/8.1.x/).